lib/Widget/evaluationForms/post-hos/vital_sign_form.dart

```diff
@@ -21,29 +21,23 @@ class VirtalSignForm extends StatefulWidget {
       : super(key: key);
 
   @override
-  _VirtalSignFormState createState() => _VirtalSignFormState();
+  _VitalSignFormState createState() => _VitalSignFormState();
 }
 
-class _VirtalSignFormState extends State<VirtalSignForm> {
+class _VitalSignFormState extends State<VitalSignForm> {
   ICalculationService _calculationService = locator<ICalculationService>();
   IFirebaseService _firebaseService = locator<IFirebaseService>();
   final _formKey = GlobalKey<FormState>();
   DateTime date = DateTime.now();
-
   double _bt;
   double _pr;
   double _rr;
   String _bp;
   int _systolic;
   int _diastolic;
+  int _o2sat;
```

Comment:
I think o2 should change to double? cuz it can be 95.6% na

Reply:
Solve in c2166ad ka